source/utilities.cc

```diff
@@ -175,16 +175,27 @@ namespace aspect
           }
         else if (Patterns::List(Patterns::Double(),1,list_of_keys.size()).match(input_string))
           {
-            // Handle the format of a comma separated list of doubles, with no keywords
-            const std::vector<double> values = possibly_extend_from_1_to_N (dealii::Utilities::string_to_double(dealii::Utilities::split_string_list(input_string)),
-                                                                            list_of_keys.size(),
-                                                                            property_name);
-
-            for (unsigned int i=0; i<values.size(); ++i)
+            if (Patterns::Double().match(input_string))
+              {
+                // Handle the format where there is only one value, assign this to all keys
+                const double value = dealii::Utilities::string_to_double(input_string);
+                for (unsigned int i=0; i<list_of_keys.size(); ++i)
+                  {
+                    parsed_map.emplace(list_of_keys[i], value);
```

Comment:
Yes, I think you are right. I change it back to the old ones and it could still pass the test. I think it should be it is already handled somehow. So I change it back to the old ones.